documentation-site/examples/data-table/basic.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import {useStyletron} from 'baseui';
+import {
+  Unstable_DataTable,
+  BooleanColumn,
+  CategoricalColumn,
+  CustomColumn,
+  NumericalColumn,
+  StringColumn,
+  COLUMNS,
+  // @ts-ignore
+} from 'baseui/data-table';
```

Comment:
Btw, it would be nice to add `index.d.ts` with `any` . It's fine not having types for unstable component but would be nice to avoid `// @ts-ignore` for consumers. They might miss errors when the component gets stable, typed but they don't remove the ignore.

Reply:
👍 will update